user_templates/overrides.cfg

```diff
@@ -63,4 +63,3 @@
 
 # [stepper_z3]
 # dir_pin: !Z3_DIR
-
```

Comment:
Can you put back this second free new line at the end? It was there on purpose to allow better separation when injecting the templates from the install script